### ISSUE TITLE
Feature Request: Creative Mode Setup

### ISSUE THREAD

**Issue by HyperMan19680** [LambdaInnovation/AcademyCraft#240]
I'd like to see a setup for Creative Mode that allows you to unlock abilities on all classes.

**Comment by WeAthFoLD** [comment]
The `/aim` command is designed for ability data editing, just use `/aim help` to see more info. For example, `/aim learn_all` unlocks all skills under current category.

**Comment by HyperMan19680** [comment]
Let me rephrase. I wanted to be able to use all the abilities for all the classes, instead of being limited to one class, while in Creative Mode as an OP.

**Comment by WeAthFoLD** [comment]
I see. I personally don't like this idea because it complicates things (the current framework is based on assumption that **one player holds one category at any time**, and why would player be able to access all skills because he/she is admin?). 

IMO you can use command to simulate what you want currently. There is something I can do to simplify the process (e.g. make all skills unlocked by default while in creative), but unlikely would I support using all skills.

In future there will be some rework in base API, that might allow 'mixin' of different categories' skills. When that starts I'll reconsider it. Sorry :-)
